pkgs/build-support/fetchgit/default.nix

```diff
@@ -45,11 +45,17 @@ in
 
 assert deepClone -> leaveDotGit;
 
-if md5 != "" then
+if rev != null && tag != null then
+  throw "fetchgit cannot take both tag and name arguments"
```

Comment:
`name` is also mutually exclusive with `tag` and `rev` right?

Reply:
Sorry, that was a mistake. `name` is independent of these arguments.